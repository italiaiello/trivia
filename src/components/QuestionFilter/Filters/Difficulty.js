import React from 'react'

const Difficulty = ({ setDifficulty }) => {

    const onDifficultyChange = e => {
        setDifficulty(e.target.value)
    }

    return (
        <select id="categories" name="categories" onChange={onDifficultyChange}>
            <option value="Any">Any</option>
            <option value="easy">Easy</option>
            <option value="medium">Medium</option>
            <option value="hard">Hard</option>
        </select>
    )
}

export default Difficulty