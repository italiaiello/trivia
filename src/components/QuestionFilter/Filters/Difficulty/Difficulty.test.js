import React from 'react'
import { shallow } from 'enzyme'
import Difficulty from './Difficulty'

it('checks if it Difficulty.js renders properly', () => {
    expect(shallow(<Difficulty />)).toMatchSnapshot()
})