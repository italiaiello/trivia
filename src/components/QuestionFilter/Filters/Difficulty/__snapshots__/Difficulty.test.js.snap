// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks if it Difficulty.js renders properly 1`] = `ShallowWrapper {}`;
